src/components/kudos/GiveKudos.jsx

```diff
@@ -16,15 +16,15 @@ const sendGiveKudosInitialState = {
   description: '',
   email: '',
   name: '',
-  amount: 0,
+  amount: BigInt(0),
 }
 
-const splitU256 = (value) => {
-  const bigIntValue = BigInt(value)
-  const BigInt128 = BigInt(2) ** BigInt(128)
+const transformInt = (value) => {
```

Comment:
can we split this out into a lib folder? 
maybe `lib/kudos.js` or `lib/utils.js`?